### PR TITLE
 [feature] Add bazelWorkspaceRootDir flag

### PR DESCRIPTION
**Pull request by tpasternak**
This PR is based on #264 by @basakkapusuzoglu

### CODE REVIEW

File: install/src/main/java/org/jetbrains/bsp/bazel/install/cli/CliOptionsProvider.kt [hunk 1]
```diff
@@ -142,6 +142,12 @@ class CliOptionsProvider(private val args: Array<String>) {
             .desc("Use bloop as the BSP server rather than bazel-bsp.")
             .build()
         cliParserOptions.addOption(useBloopOption)
+
+        val bazelWorkspaceRootDirOption = Option.builder(BAZEL_WORKSPACE_ROOT_DIR_OPT)
+            .longOpt("bazel workspace root dir")
```
Comment (abrams27) [inline install/src/main/java/org/jetbrains/bsp/bazel/install/cli/CliOptionsProvider.kt:147]:
```suggestion
            .longOpt("bazel_workspace")
```

**Review by abrams27 — CHANGES_REQUESTED** [review]

**Comment by tpasternak** [comment]
btw, `//e2e:BazelBspRemoteJdkTest` seems to be flaky

**Comment by tpasternak** [comment]
Rebased and force-pushed on current master

File: install/src/main/java/org/jetbrains/bsp/bazel/install/cli/CliOptionsProvider.kt [hunk 2]
```diff
@@ -165,6 +165,13 @@ class CliOptionsProvider(private val args: Array<String>) {
             .desc("Use bloop as the BSP server rather than bazel-bsp.")
             .build()
         cliParserOptions.addOption(useBloopOption)
+
+        val bazelWorkspaceRootDirOption = Option.builder(BAZEL_WORKSPACE_ROOT_DIR_OPT)
+            .longOpt("bazel_workspace")
+            .hasArg()
+            .desc("Path to directory where workspace is located.")
```
Comment (abrams27) [inline install/src/main/java/org/jetbrains/bsp/bazel/install/cli/CliOptionsProvider.kt:172]: lets add what is the default - the `-d` and if `-d` is not specified then it's a pwd

**Review by abrams27 — COMMENTED** [review]
thanks a lot!

please update the changelog

**Review by abrams27 — CHANGES_REQUESTED** [review]

**Review by abrams27 — APPROVED** [review]

**Pull request merged**